collect.py

```diff
@@ -170,6 +172,10 @@ def do_something(driver, elem_attributes=None):
             driver.execute_script("arguments[0].value = '#ff0000'", elem)
         else:
             raise Exception('Unsupported input type: %s' % input_type)
+    elif elem.tag_name == 'select':
+        for option in elem.find_elements_by_tag_name('option'):
```

Comment:
There's a bit of a problem. What if the order of the elements in the browsers is not the same?
We should click on exactly the same elements in both browsers, so we should make note of the attributes of the option element on which we have clicked on the first browser so that we can repeat the same action on the second browser.
Anyway, we can fix this in a follow-up.